.github/workflows/cli-scripts-deploy-safe-rollout-kubernetes-online-endpoints.yml

```diff
@@ -1,45 +0,0 @@
-name: cli-scripts-deploy-safe-rollout-kubernetes-online-endpoints
```

Comment:
is there any content change for those workflows? Only rename?
There was a shell to run workflows that start with cli and sdk for cli/sdk team to do testing, if they still use this way, our workflows will be ignored for their testing before publish.

Reply:
Thanks for the reminder. Changes about inference workflow have been revoked.